mailq.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Description: Expose the size of the mail queue from mailq command.
+#
+# Author: Thore Guentel <thore.guentel@tangogolf.de>
+#
+# Results:
+# * 0 and more is the count of the mailq objects
+# * -1 means mailq command is not found
+# * -2 means error at checking the mailq
+#
+
+METRIC="mailq_size_total"
+HELPTXT="# HELP $METRIC Count of Mails in the Queue"
+TYPETXT="# TYPE $METRIC gauge"
+
+MAILQCMD=$(which mailq)
+if [ "$MAILQCMD" != "" ]
```

Comment:
Just for future reference: [Bash sets the status to 127 for unavailable commands](https://www.gnu.org/software/bash/manual/html_node/Exit-Status.html).